libs/components/indicators/src/lib/modules/expansion-indicator/expansion-indicator.component.ts

```diff
@@ -6,7 +6,10 @@ import { Component, Input } from '@angular/core';
  */
 @Component({
   selector: 'sky-expansion-indicator',
-  styleUrls: ['./expansion-indicator.component.scss'],
```

Comment:
I'm new to this; are we keeping the old stylesheet (why not delete)?

Reply:
It looks like I missed deleting the original stylesheet after resolving some merge conflicts. It's gone now.

Reply:
Cool.